### PR TITLE
Captain's Rapier

### PR DESCRIPTION
**Pull request by dwinters99**
While playing site manager, I thought to myself: "Damn, this job sucks if I don't have an impressive sword I can use to separate me from the proletariat."
I then added an impressive sword and sumptuary law.

- Ports the captain's rapier from /tg/ and adds it to the captain's locker
- General sword sheathe (with special icon) that can hold assorted other swords (almond spawn only)

### CODE REVIEW

**Comment by dwinters99** [comment]
Swore I tested it but I guess I didn't. Whoops.

**Comment by Wickedtemp** [comment]
Don't know if you'd be able to do much with it. I'm pretty sure it'd be against Regs to have it out or do anything with it unless shit's hitting the fan, at which point the CD shouldn't be fighting anyway.

File: code/game/objects/items/weapons/melee/misc_vr.dm [hunk 1]
```diff
@@ -0,0 +1,16 @@
+/obj/item/weapon/melee/rapier
+	name = "rapier"
+	desc = "A gleaming steel blade with a gold handguard and inlayed with an outstanding red gem."
+	icon = 'icons/obj/weapons_vr.dmi'
+	icon_state = "rapier"
+	item_icons = list(
+		slot_l_hand_str = 'icons/mob/items/lefthand_melee_vr.dmi',
+		slot_r_hand_str = 'icons/mob/items/righthand_melee_vr.dmi',
+		)
+	force = 25
```
Comment (Andromeda-K22) [inline code/game/objects/items/weapons/melee/misc_vr.dm:10]: This probably should have force in line with something that's meant to be ceremonial.

**Comment by Heroman3003** [comment]
Yeah, I'm unsure about paper-pusher managerial role getting a super robust melee weapon. Seems to be sending extremely wrong messages.

**Comment by lm40** [comment]
idk to me CAPTAIN sounds like a job that is thematic to have a sabre for, but MANAGER is not really a job thats giving me the same vibes

**Comment by Meghan-Rossi** [comment]
>The high damage is actually a copy-paste error that I didn't notice until now. It's meant to have 15 force. (crowbar has 7.)

That's still as much as a harmbaton or telebaton, and the sharpness makes it deadlier.

**Comment by dwinters99** [comment]
> idk to me CAPTAIN sounds like a job that is thematic to have a sabre for, but MANAGER is not really a job thats giving me the same vibes

I do actually slightly agree with this, but I feel that even a space station would still inherit the traditions of seafaring much like a ship might. Also, from an in-universe perspective, I feel a site manager might be treated with equal respect to that of a ship captain and therefore carry on the ceremony of it. This is compounded with the ceremonial garb in the SM's office. Why not have a ceremonial sword to go with the parade outfit?

> That's still as much as a harmbaton or telebaton, and the sharpness makes it deadlier.

It IS still a sword, ceremonial or not. Besides, the SM already gets a telebaton, so having something with slightly higher potency along with the fact that an SM should see virtually no combat or reason to use it, makes it in-line with the general power balance of the available items. Also, if the SM **REALLY** wanted to hurt someone, they get an energy gun.

**Comment by Novacat** [comment]
I am going to tentatively merge this and see where it goes. 15 force is not super robust for a melee weapon, and if the SM really wanted to hurt someone they would probably just grab an explorer or the pathfinder's machete which are a lot more robust (and start off made of plasteel). But if I see SMs rushing mobs and trying to stabby them then I will probably nerf this further.

**Pull request merged**